README.md

```diff
@@ -52,20 +277,13 @@ code_generator.write_code()
 
 # Run the generated code and verify the output xml against the source
 # Only: Python, PHP, Node
+# TODO: ADD other helper libraries to code_generator.verify?
```

Comment:
This TODO feels out of a place in a README -- maybe this could be an issue or a ticket, or a TODO in the generator code itself?

Reply:
lol whoops! You're right.